intelmq/bots/BOTS

```diff
@@ -653,6 +103,45 @@
                 "xmpp_server": "xmpp server",
                 "xmpp_user": "xmpp username"
             }
+        },
+        "AlienVault OTX": {
```

Comment:
Why has this been moved from the top to the bottom? This list should be sorted (also see bin/rewrite_config_files.py)
EDIT: Haven't seen the discussion yet.